mealie-recipes/mealie
# fix: mealie docs 404 page broken styling

## What type of PR is this?



- bug

## What this PR does / why we need it:

- Fixes a bug in the 404 template that broke styling for documentation server 404 page.



## Which issue(s) this PR fixes:

Fixes #2841 



## Testing

- Run the docs server
- Load a nonexistent page
- Observe that styling is no longer broken
